tasks/app.js

```diff
@@ -0,0 +1,19 @@
+var endOfLine = require('os').EOL;
+process.stdin.setEncoding('utf8');
+
+process.stdin.on('readable', () => {
```

Comment:
Consider writing separate function to contain you callback logic for std events.